hunting-buddy.lic

```diff
@@ -76,6 +78,16 @@ class HuntingBuddy
     bput("stow my #{description}", 'You put')
   end
 
+  def check_prebuffs
+    return unless @settings@settings.pre_hunt_buff 
+    return unless !@settings.waggle_sets.nil?
```

Comment:
This could be simplified to `return if @settings.waggle_sets.nil?` or even `return unless @settings.waggle_sets`.